README.md

```diff
@@ -118,13 +118,16 @@ $ cd terraform
 $ cp example.tfvars terraform.tfvars
 ```
 
-Fill in your username as the `cert_owner`:
+Fill in your username as the `cert_owner` via:
 
 ```sh
 $ sed -i "s/FIXME_USER/$USER/" terraform.tfvars
 ```
 
-Check terraform.tfvars and change any values you would like to change. Note that the values given in examples.tfvars is NOT completely AWS free tier eligible, as they include t2.small and t2.medium instances. We do not recommend using t2.micro instances, as they were unable to compile solidity during testing.
+Check terraform.tfvars and change any values you would like to change:
+- You may need to modify the network ID to one which is not in use.  
+- We assume your home directory is at `/Users/$USER` for locating your keyfiles, you will need to update the `public_key_path` (and `private_key_path`, if present) if that isn't the case.
```

Comment:
We expect that path is correct for OS X in particular. Just mention that so there's less confusion.